apps/dashboard/src/data/editor/reducer.js

```diff
@@ -8,9 +8,13 @@ import { combineReducers } from '@wordpress/data';
  */
 import {
 	EDITOR_CONTENT_CHANGED_SET,
+	EDITOR_CONTENT_RESTORE,
 	EDITOR_CONTENT_SAVE,
 	EDITOR_CONTENT_SAVE_ERROR,
 	EDITOR_CONTENT_SAVE_SUCCESS,
+	EDITOR_CONTENT_UPDATE,
+	EDITOR_TITLE_SET,
+	PROJECT_UPDATE,
```

Comment:
One action I'd add is `EDITOR_INIT` - which would 'reset' this tree when `<Editor />` is first loaded or `projectId` changes.  
It might also be useful to have a `state.editor.projectId`.

So you'd initialize the editor with something like `initializeEditor( projectId )`, that way we can later also reference that in what project the changes should be saved to.

Reply:
What?!

Reply:
This isn't actually necessary right now, although having current `projectId` inside `state.editor` would be nice.  

What I'm referring to is a situation down the line, you edit one project, exit, eventually make your way to editing another one, or even the same one - we need to make sure `state.editor` is reset to the defaults/empty before that second editor gets loaded. So I guess a general `initializeEditor( projectId )` would take care of any setup loading a new project might require - but most of it is not a problem for now anyway so maybe let's not overthink this too much.